composer.json

```diff
@@ -13,7 +13,7 @@
   },
   "require": {
     "symfony/yaml": "~2.6",
-    "guzzlehttp/guzzle": "~5.0",
```

Comment:
I don't know why this was included. We weren't using it anywhere as far as I checked.